package.json

```diff
@@ -4,7 +4,7 @@
   "description": "a knex mock adapter for simulating a db during testing",
   "main": "index.js",
   "scripts": {
-    "test": "./node_modules/lab/bin/lab -v -I __BluebirdErrorTypes__"
+    "test": "./node_modules/lab/bin/lab -v -I __BluebirdErrorTypes__,Set,Intl,Map"
```

Comment:
I believe this was only necessary because I'm using node 5.11. I can remove if it's excess.